Resources/doc/reference/architecture.rst

```diff
@@ -354,6 +354,32 @@ Then, you have to set the CommentAdmin ``parentAssociationMapping`` attribute to
         }
     }
 
+To display the CommentAdmin extend the menu in your ``PostAdmin`` class:
+
+.. code-block:: php
+
+    <?php
+    namespace Sonata\NewsBundle\Admin;
+
+    class PostAdmin extends Admin
+    {
+        // ...
+
+        protected function configureSideMenu(MenuItemInterface $menu, $action, AdminInterface $childAdmin = null)
+        {
+            if (!$childAdmin && !in_array($action, array('edit', 'show'))) { return; }
```

Comment:
Indeed, didn't see it.

According to coding standards, this should not be one lined.